### ISSUE TITLE
Solution for: 'ValueError: 'src/cld3.pyx' doesn't match any files' 

### ISSUE THREAD

**Issue by simonsigre** [CIRCL/AIL-framework#535]
When installing AIL-framework on `Ubuntu 20.04.1 LTS` you may find that Flask wont start (and as such your `127.0.0.1:7000` listener won't come up).

Switching to the Flask `screen` you note there are a number of errors reporting `cld3` missing, attempting a manual install reports the following;
```
  Using cached cld3-0.2.2.tar.gz (593 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-tr_dqwdo/cld3/setup.py'"'"'; __file__='"'"'/tmp/pip-install-tr_dqwdo/cld3/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-tr_dqwdo/cld3/pip-egg-info
         cwd: /tmp/pip-install-tr_dqwdo/cld3/
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-tr_dqwdo/cld3/setup.py", line 98, in <module>
        ext_modules=cythonize([ext]))
      File "/usr/local/lib/python3.8/dist-packages/Cython/Build/Dependencies.py", line 965, in cythonize
        module_list, module_metadata = create_extension_list(
      File "/usr/local/lib/python3.8/dist-packages/Cython/Build/Dependencies.py", line 815, in create_extension_list
        for file in nonempty(sorted(extended_iglob(filepattern)), "'%s' doesn't match any files" % filepattern):
      File "/usr/local/lib/python3.8/dist-packages/Cython/Build/Dependencies.py", line 114, in nonempty
        raise ValueError(error_msg)
    ValueError: 'src/cld3.pyx' doesn't match any files
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

The solution is to install the Python Bindings (`https://github.com/bsolomon1124/pycld3`), I did this by swapping to the `Flask` screen and issuing the following
```
sudo pip3 install pycld3
```

**Comment by simonsigre** [comment]
This appears to be "fixed" as part of https://github.com/CIRCL/AIL-framework/commit/e3800aa36a524936185334a752bc7d4497325733 ... however I have deployed this in the past few days and that dependency wasn't met and had to be re-run.

**Comment by Terrtia** [comment]
Fixed in v5.3
cld3 has been replaced by gcld3.
